com.vagueentertainment.Framed.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.vagueentertainment.Framed.desktop</id>
+  <name>Framed</name>
+  <summary>Digital Picture frame for your NFTshowroom art and personal work</summary>
+  <description>
+    <p> Early version of Framed
+    </p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <url type="homepage">https://vagueentertainment.com/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://godotengine.org/storage/app/media/3.0%20release/gltf.png</image>
```

Comment:
Looks like a screenshot from Godot to me!

Reply:
Hah, I new I forgot something.